controllers/operator/authentication_controller.go

```diff
@@ -377,20 +391,6 @@ func (r *AuthenticationReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return subreconciler.Evaluate(subResult, err)
 	}
 
-	// perform any migrations that may be needed before Deployments run
```

Comment:
Hi @rwhundley , should we also need to move `ensureDatastoreSecretAndCM` function call ?

Reply:
Yeah, missed that - thanks!